docs/DeveloperGuide.md

```diff
@@ -381,9 +383,62 @@ The _sequence diagram_ below shows how the components interact with each other w
 
 ![Sequence diagram of sort command](images/SortSequenceDiagram.png)
 
-Note that
+### Home Page UI
+
+##### Overview
+
+`HomeCard` is a new feature in the UI component designed to provide users with a quick overview of crucial information immediately upon application launch. It serves as a dynamic dashboard, displaying the current date and time, the total number of contacts, a summary of financial transactions, and a list of contacts available for the day.
+
```

Comment:
Maybe can add screenshot to show what HomeCard is? You can draw like a box and then label which is the HomeCard